lambdaclass/mirra_backend
# [GH-753] Fix bot spawning

## Motivation

We want to be able to decide if we spawn additional players as bots in matches 

## Summary of changes

Added new environmental var to not spawn bots if it's on false

## How to test it?

Start the game and run a match, you should be alone. Then run `System.put_env("SPAWN_BOTS", "true")` and if you enter another match you should see bots again 

## Checklist
